gunicorn/glogging.py

```diff
@@ -226,7 +231,11 @@ def setup(self, cfg):
                     self.access_log, cfg, self.syslog_fmt, "access"
                 )
 
-        if cfg.logconfig:
+        if dictConfig and cfg.logconfig_dict:
+            config = CONFIG_DEFAULTS.copy()
+            config.update(cfg.logconfig_dict)
+            dictConfig(config)
```

Comment:
We probably need to catch the following exceptions and raise `RuntimeError(message_from_catched_exception)` like we already did for `logconfig` in line 247:

> If an error is encountered during configuration, this function will raise a ValueError, TypeError, AttributeError or ImportError with a suitably descriptive message.